db/migrate/20240626152030_add_accept_a_level_equivalencies_to_course.rb

```diff
@@ -0,0 +1,5 @@
+class AddAcceptALevelEquivalenciesToCourse < ActiveRecord::Migration[7.1]
+  def change
+    add_column :course, :accept_a_level_equivalencies, :boolean
```

Comment:
After I opened the PR I realise the other fields for GCSE and degrees are *_equivalency for this field. Let me know if I should change now.

Reply:
Done